evap/rewards/templates/rewards_reward_point_redemption_events.html

```diff
@@ -7,10 +7,14 @@
 
 {% block content %}
     {{ block.super }}
-
-    <p>
-        <a href="{% url 'rewards:reward_point_redemption_event_create' %}" class="btn btn-dark btn-sm">{% trans 'Create new event' %}</a>
-    </p>
+    <div class="row mb-2 align-items-center">
+        <div class="col">
+            <a href="{% url 'rewards:reward_point_redemption_event_create' %}" class="btn btn-dark btn-sm">{% trans 'Create new event' %}</a>
+        </div>
+        <div class="col" style="text-align: right">
```

Comment:
There is a bootstrap class for this style already:
```suggestion
        <div class="col text-end">
```